.github/workflows/docker.yml

```diff
@@ -0,0 +1,49 @@
+name: docker
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      # build for every branch push
+      - "**"
```

Comment:
Do we really want to build a docker image on every push?

Reply:
To allow branches like `growl` this currently is required as there is no convention on branch naming for new features or updates